old-ui/app/domainsPage.js

```diff
@@ -0,0 +1,63 @@
+const inherits = require('util').inherits
+const Component = require('react').Component
+const h = require('react-hyperscript')
+const connect = require('react-redux').connect
+const actions = require('../../ui/app/actions')
+
+module.exports = connect(mapStateToProps)(DomainsScreen)
+
+function mapStateToProps (state) {
+  return {}
+}
+
+inherits(DomainsScreen, Component)
+function DomainsScreen () {
+  Component.call(this)
+}
+
+DomainsScreen.prototype.render = function () {
```

Comment:
I know that the repo has this all the same, but can we use the classic react syntax instead? it's easier to understand and maintain i think, no prototypes involved, here it's an example:

```
class DomainScreen extends React.Component {
  navigateTo() {
    global.platform.openWindow({ url })
  }
  render() {
    //here your render code
  }
}
```

Reply:
Ok, i was just trying to use the same conventions as the old project haha. Let me change this and let's start using the react syntax as you say.

Reply:
Thanks, i think it will be better for us in the future, also we can differenciate our code from the original code :wink:

Reply:
I agree with @JONAF2103 here